tests/plugins/tabletools/tabletools.js

```diff
@@ -262,6 +262,33 @@
 				bot.setHtmlWithSelection( source );
 				assert.beautified.html( expected, bot.getData( true ) );
 			} );
+		},
+
+		'test merge cells with a rowspan should create undo step': function() {
+
+			bender.editorBot.create( {
+				name: 'editor_merge_rowspanned_cells',
+				config: {
+					plugins: 'undo,table,tableselection'
+				}
+			}, function( bot ) {
+				bender.tools.ignoreUnsupportedEnvironment( 'tableselection' );
+
+				var editor = bot.editor;
+
+				bender.tools.testInputOut( 'merge-rowspanned-cells', function( source, expected ) {
+					bot.setHtmlWithSelection( source );
+					bot.execCommand( 'cellMerge' );
+
+					var output = bot.getData( true ),
+						undo = editor.getCommand( 'undo' );
+
+					output = output.replace( /\u00a0/g, '&nbsp;' );
```

Comment:
This line of code seems to be unnecessary. Why do we need this replacement? The test looks fine without it.

Reply:
Yes, it looks like it's not needed. Removed